extensions/github/src/commands.ts

```diff
@@ -20,6 +20,16 @@ async function copyVscodeDevLink(gitAPI: GitAPI, useSelection: boolean) {
 	}
 }
 
+async function openVscodeDevLink(gitAPI: GitAPI): Promise<vscode.Uri | undefined> {
+	try {
+		const permalink = getPermalink(gitAPI, true, 'https://vscode.dev/github');
```

Comment:
Should we make this insiders aware? Ie, when in Insiders, `https://insiders.vscode.dev/github` is used.

Reply:
Same for `copyVscodeDevLink` I guess.

Reply:
I recall @lszomoru had the suggestion that we give the user the option to choose between stable and insiders, i.e. display a quickpick (and optionally offer a setting to remember the preference), thoughts on that?